metabase/metabase
# Visually improve time-series chrome component

### Description

This PR changes how the TimeSeriesChrome component look and feel like in subtle, yet impactful ways.
1. It stacks the operator picker on top of the actual filter pickers
2. It reuses the same CurrentDate picker logic that we're using in other date pickers in the app

The component was very wide in the past for the Interval date pickers, such as "Previous" or "Next" because the operator picker was rendered in a row next to the actual filter picker.

This PR completes the Milestone 3 of #44096, and it resolves #44096.

### How to verify
Tests already exist for all these interactions so they should pass in CI.

If you want to manually play with this, follow these steps:
1. Open Orders table and summarize it by Count, broken down by Created At: month
2. Time-series chrome component should be visible in the lower part of the screen

From here you can choose either
- "Previous" or "Next", or
- Current

### Demo
**Interval Picker**

Before
![image](https://github.com/metabase/metabase/assets/31325167/4406b34d-dd31-4b6f-aa4f-7136f5804cf1)

After
![image](https://github.com/metabase/metabase/assets/31325167/e91c178c-f337-467b-bdd8-368378bdef7b)

**Current Picker**

Before
![image](https://github.com/metabase/metabase/assets/31325167/9fb350b5-fb18-4b00-b457-e8ea56e14429)

After
![image](https://github.com/metabase/metabase/assets/31325167/1b2c0aff-6cc0-4a32-896d-c2b48babd3c8)


### Checklist


### Additional Notes
Figma conversation that sparked this: https://www.figma.com/design/ofKULr1X7TQf5MZCkbJZvx?node-id=4030-3370#801341512
